src/Refinery/String/EstimatedReadingTime.php

```diff
@@ -66,6 +66,10 @@ private function calculate(string $text) : int
             foreach ($textNodes as $textNode) {
                 /** @var \DOMText $textNode */
                 $wordsInContent = array_filter(preg_split('/\s+/', $textNode->textContent));
+
+                $wordsInContent = array_filter($wordsInContent, function (string $word) {
+                    return preg_replace('/^\pP$/', '', $word) !== '';
```

Comment:
Sry, I overlooked this, but: Why is there an additional P in the middle of the regexp?

Reply:
This means `Any Kind of Punctuation Character`, see here https://www.regular-expressions.info/posixbrackets.html#class and test here https://regex101.com/r/LDxtKd/2

Reply:
Thx!

Reply:
`\p{P}` is equivalent to `\pP`